tsl/src/continuous_aggs/invalidation.c

```diff
@@ -775,7 +775,7 @@ move_invalidations_from_hyper_to_cagg_log(const CaggInvalidationState *state)
 	forboth (lc1, all_caggs->mat_hypertable_ids, lc2, all_caggs->bucket_widths)
 	{
 		int32 cagg_hyper_id = lfirst_int(lc1);
-		int64 bucket_width = *(int64 *) lfirst(lc2);
+		int64 bucket_width = DatumGetInt64((Datum) lfirst(lc2));
```

Comment:
You are storing into these lists using `DatumGetPointer(Int64GetDatum(int64))` but when reading you do `DatumGetInt64(elem)`. Shouldn't you add a `PointerGetDatum` when reading it? I do see the cast, but that just happen to be the same as the macro, so it's probably better to be consistent here.